src/App.js

```diff
@@ -1,12 +1,73 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import Pokemon from './components/Pokemon';
 import './App.css';
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [data, setData] = useState([]);
+    const [isLoaded, setIsLoaded] = useState(false);
+    const [offset, setOffset] = useState(0);
+
+    async function getData() {
+        fetch("https://pokeapi.co/api/v2/pokemon?limit=20&offset=" + offset)
+            .then(res => res.json())
+            .then(
+                (result) => {
+                    setData(result.results);
```

Comment:
Je zou hier nog spreading kunnen toepassen
```suggestion
                ({ results }) => {
                    setData(results);
```